.default.env

```diff
@@ -1,6 +1,7 @@
 REACT_APP_BEE_URL=http://localhost:1633
 REACT_APP_BEE_DEBUG_URL=http://localhost:1635
 REACT_APP_RPC_URL=http://localhost:9545
+REACT_APP_FAIROS_URL=http://localhost:9090
```

Comment:
please add to readme 
1. link to fairos repo 
2. link to installation fairos instructions 
3. specify why it is required

Reply:
add run fdp-play with another command: 'fdp-play start --fresh --blockchain-image fairdatasociety/fdp-contracts-blockchain:1.1.1'